delfin/drivers/utils/ssh_client.py

```diff
@@ -240,11 +240,12 @@ def put(self, conn):
             return
         super(SSHPool, self).put(conn)
 
-    def do_exec(self, command_str):
+    def do_exec(self, command_str, is_check=True):
         result = ''
         try:
             with self.item() as ssh:
-                utils.check_ssh_injection(command_str)
+                if is_check:
```

Comment:
These two if statements can be moved outside the with statement.

Reply:
ok